templates/main-org.template.yaml

```diff
@@ -525,7 +525,7 @@ Parameters:
   Section3CloudTrailBucketPathExpression:
     Type: String
     Description: "The path expression must match the folder structure for CloudTrail logs (e.g., AWSLogs/*/CloudTrail/*)."
-    Default: "CloudTrail/AWSLogs/*/CloudTrail/*"
+    Default: "CloudTrail/AWSLogs/*/*/*"
```

Comment:
Can we just change this to `CloudTrail/AWSLogs/*` ?

Reply:
Can change it

Reply:
changed